docker_images/sklearn/tests/test_api.py

```diff
@@ -5,10 +5,17 @@
 from app.main import ALLOWED_TASKS, get_pipeline
 
 
-# Must contain at least one example of each implemented pipeline
-# Tests do not check the actual values of the model output, so small dummy
-# models are recommended for faster tests.
-TESTABLE_MODELS: Dict[str, str] = {"tabular-classification": "julien-c/wine-quality"}
+# Must contain at least one example of each implemented pipeline Tests do not
+# check the actual values of the model output, so small dummy models are
+# recommended for faster tests. Note: make sure all models require
+# scikit-learn=1.0.2, and if you change that, make sure README.md of sklearn
+# folder is also updated.
+TESTABLE_MODELS: Dict[str, str] = {
+    "tabular-classification": {
```

Comment:
It might make sense to have a couple extra tests for some of the other conditions in the pipeline
* What if file in config.json is invalid (e.g. no sklearn key)
* Missing config.json

The following test structure might be useful here https://github.com/huggingface/api-inference-community/blob/main/docker_images/speechbrain/tests/test_api.py#L11

Reply:
Yep, will get a space for those repos and add the tests

(cc @merveenoyan this is where we need those repositories we just talked about offline)